monochart/values.yaml

```diff
@@ -712,3 +712,12 @@ networkPolicy:
   #         protocol: TCP
 
 
+
+# https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+hpa:
+  enabled: false
+#   autoscaling:
+#     minReplicas: 1
+#     maxReplicas: 10
+#     targetCPUUtilizationPercentage: 90
+#     targetMemoryUtilizationPercentage: 90
```

Comment:
Should these stay commented? Will defaults be set if they are? What about the use case where someone only specifies `enabled: true` - is that supported?

Reply:
I can't remember why we comment the values. I think it's because those would need to be added to catalog file but unsure. Pinging @dalefrancum any chance you recall why we did this?